source/fundamentals/data-formats/document-data-format-bson.txt

```diff
@@ -13,11 +13,82 @@ Document Data Format: BSON
 Overview
 --------
 
-- Definition of BSON format, where it's used, why it's used.
-- Link to more info on BSON
-- What information you can find in the following sections
+This guide explains the BSON data format, how MongoDB uses it, and how to
+install the BSON library independently of the MongoDB driver.
 
-Installation
-------------
+BSON Data Format
+----------------
 
-- Instructions on how to add the dependency with Maven or Gradle
+**BSON**, or Binary JSON, is the data format that MongoDB uses to organize
+and store data. This data format includes all JSON data structure types and
+adds support for binary types. BSON is a serialized form of JSON optimized for 
+scan speed and transmission over the network. For more information on the
+types, see the :manual:`BSON Types </reference/bson-types>` server manual page.
+
+The binary format is not human-readable, but you can use the
+:ref:`Java BSON library <install-bson-library>` to convert it to a JSON
+representation. You can read more about the relationship between these
+formats in our article on :website:`JSON and BSON </json-and-bson>`.
+
+MongoDB and BSON
+----------------
+
+MongoDB uses the BSON data format for reasons including:
```

Comment:
It seems strange to include this in the Java driver docs directly, rather than linking out to something common like https://docs.mongodb.com/manual/reference/bson-types/.

Reply:
The server manual page is linked in the prior section. I'm not sure the content on that page expresses what some of the reasons are for using BSON with MongoDB.
I wasn't able to find a good reference on this amongst the server manual or other docs, but wanted to provide some information that helps a user understand a particular choice that was made, which I've found some users asking about in user studies.
If you think this section makes the docs less concise or if there's any inaccurate information, let me know and I'll remove it.

Reply:
I would hesitate to put this text here.  If you want to add something to the main BSON page and have it reviewed in that context I'd be fine with that, but it seems unwise to have it in the java driver docs directly

Reply:
Ok sure. Removed.